docs/source/gbd2019_models/causes/ihd/index.rst

```diff
@@ -116,331 +112,224 @@ Assumptions and Limitations
-
-.. list-table:: Data Sources
-   :widths: 20 25 25 25
+.. list-table:: Data Sources and Definitions
+   :widths: 10 10 20 20
    :header-rows: 1
-   
-   * - Measure
-     - Sources
-     - Description
-     - Notes
-   * - prevalence_cid
-     - 
-     - 
-     - 
-   * - birth_prevalence_cid
-     - 
-     - 
-     -
-   * - deaths_cid
-     - 
-     - 
-     - 
-   * - population
-     - 
-     - 
-     - 
-   * - sequelae_cid
-     - 
-     - 
-     - 
-   * - incidence_rate_cid
-     - 
-     - 
-     - 
-   * - remission_rate_m1594
-     - 
-     - 
-     - 
-   * - disability_weight_s{`sid`}
-     - 
-     - 
-     - 
-   * - prevalence_s{`sid`}
-     - 
-     - 
-     - 
 
-An example, that contains common sources for the measures,
-
-.. list-table:: Data Sources
-   :widths: 20 25 25 25
-   :header-rows: 1
-   
-   * - Measure
-     - Sources
+   * - Variable
+     - Source
      - Description
      - Notes
-   * - prevalence_cid
+   * - prevalence_c493
      - como
-     - Prevalence of cause
-     - 
-   * - birth_prevalence_cid
-     - como
-     - Birth prevalence of cause
+     - prevalence of ischemic heart disease
      -
-   * - deaths_cid
+   * - deaths_c493
      - codcorrect
-     - Deaths from cause
-     - 
+     - Count of deaths due to ischemic heart disease
+     -
    * - population
      - demography
-     - Mid-year population for given age/sex/year/location
-     - 
-   * - sequelae_cid
-     - gbd_mapping
-     - List of sequelae
-     - 
-   * - incidence_rate_cid/mid
-     - como/dismod
-     - Incidence rate for cause
-     - 
-   * - remission_rate_cid/mid
-     - como/dismod
-     - Remission rate for cause
-     - 
-   * - disability_weight_s{`sid`}
+     - Mid-year population for given sex/age/year/location
+     -
+   * - prevalence_s{sid}
+     - como
+     - Prevalence of sequela with id {id}
+     -
+   * - disability_weight_s{sid}
      - YLD appendix
-     - Disability weight of sequela with id `sid`
-     - 
-   * - prevalence_s{`sid`}
+     - Disability weight of sequela with id {id}
+     -
+   * - incidence_493
      - como
-     - Prevalence of sequela with id `sid`
-     - 
+     - Incidence of ischemic heart disease
+     -
+   * - emr_m15755
+     - dismod-mr
+     - excess-mortality rate of post-MI ischemic heart disease
+     -
+   * - emr_m24694
+     - dismod-mr
+     - excess-mortality rate of MI due to ischemic heart disease
+     -
+   * - emr_m1817
+     - dismod-mr
+     - excess-mortality rate of angina due to ischemic heart disease
+     -
+   * - acute-sequelae
+     - model assumption
+     - {s378, s379}
+     -
+   * - post-mi-sequelae
+     - model assumption
+     - {s383, s384, s385, s1040, s5726}
+     -
+   * - angina-sequelae
+     - model assumption
+     - {s380, s381, s382, s953}
+     -
 
 
 Validation Criteria
 +++++++++++++++++++
 
+.. todo::
+
+  Add in validation criteria
```

Comment:
definitely include checks of the age-/sex-specific incidence, prevalence, excess mortality, cause-specific mortality, and all-cause mortality rates --- validate that the simulation comes up with rates within X% of the GBD estimates.